db/README.md

```diff
@@ -16,48 +13,55 @@ This `Dockerfile` can be built independently from any other in this project.
 ## Build
 
 ```sh
-$ docker build -t trinitycore-db .
+docker build -t trinitycore-db .
 ```
 
-Builds a mysql database image for TrinityCore. The resulting docker image has a
-custom entrypoint for executing various tasks and running the mysql database.
-
-### data
+Builds a mariadb database image for TrinityCore that contains the current create_sql script for 3.3.5
+Rather than create multiples entry point we will just copy the file and go inside the container to init it.
+The -p 3306:3306 will expose the port of the db container, so you can connect to it via an external SQL tool like dbeaver.
 
-The data command creates a [data-only][] container. A data-only container can
-be used to persist and share data across multiple containers. It is assumed you create a data-only container first before running anything else.
+## Run the db and init it once
 
 ```sh
-$ docker run --name tc-mysql-data trinitycore-db data
+docker run -d --name tc-dbserver -p 3306:3306 --restart=always -v TC-db:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=password trinitycore-db
 ```
 
-`tc-mysql-data` is now a container where all mysql data can persist.
-
-[data-only]: https://docs.docker.com/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container
-
-### init
+This will run the container and mark it for restart once your computer restart, else you will have to run this command each time.
 
-Initializes mysql it with the base [TDB](http://collab.kpsn.org/display/tc/Databases+Installation).
+Once the container is started, we will attach to it and run the create script. 
 
 ```sh
-$ docker run --rm -ti --volumes-from tc-mysql-data -e MYSQL_ROOT_PASSWORD=password trinitycore-db init
+docker exec -it tc-dbserver /bin/bash
 ```
 
-### mysqld
-
-Starts the mysql database. Note the use of `--volumes-from` to pull in the persisted database data from the `tc-mysql-data` container, as well as the `-d` for daemon mode. The name used, `tc-dbserver`, will be used to later link this container to the world and auth servers.
+Once inside the container, run the following to init the db. 
+The first sed command will replace the default password (trinity currently) by your own secure password.
+The second will make the user trinity able to connect from any ip address rather than just localhost.
 
 ```sh
-$ docker run -d --name tc-dbserver --volumes-from tc-mysql-data -e MYSQL_ROOT_PASSWORD=password trinitycore-db mysqld
+cd /etc/db/sql
+sed -i "s/IDENTIFIED BY '.*'/IDENTIFIED BY 'mysecurepassword'/" create_mysql.sql
```

Comment:
@RezoApio I think this is the line that's causing the worldserver to fail for the first time. It seems the default password for the trinity user is `trinity`. If I set this value to `trinity`, I'm able to start to worldserver.

Do you know off-hand how to configure the worldserver to use a different password? Is that via a conf? We should make the documentation clear that if you change the password at this step, you'll need to change something else before running worldserver.

Reply:
@josefalcon the trinity password is set up from the create_mysql.sql file from the db docker file.
I suppose I can add some better documentation around that point. Will do that when back home

Reply:
Right, but what password is the worldserver using when it attempts to connect to the db? It needs to be the same as what is set here, but it's not clear to me where it's set for the worldserver...

Reply:
At the DB container level,
the first docker run will create the ROOT password for mysql.
the second will allow you to change the default trinity password for the trinity user.

and at the docker container level, I have fogotten to allow a password change.
This is between line 35 and 37 of the entrypoint.sh script. I should allow for passing the variable with a -e param....

Good catch! I will fix this also and repull from my new branch.